chit_chat/viewsets.py

```diff
@@ -22,3 +23,16 @@ def get_queryset(self):
             'members',
         )
         return qs
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
```

Comment:
should you do this in the serializer? in `validate(self, attrs)`?

Reply:
(that way you don't have all this boilerplate n such)

Reply:
Ah good call. Can to it in `def create` there probably.